### PR TITLE
Updating requests to revert the undergraduate student message

### PR DESCRIPTION
**Pull request by carolyncole**

### CODE REVIEW

File: app/views/catalog/_home_text.html.erb [hunk 1]
```diff
@@ -34,10 +34,10 @@
   </div>
   <div class="covid19-text">
     <%= link_to "https://library.princeton.edu/phased-planning-updates" do %>
-      <h3>New Services: Pick-up and Digitization</h3>
+      <h3>New Services: Study-Browse Service, Pick-up, and Digitization</h3>
     <% end %>
     <p>
-    Firestone and most branch libraries are now offering pick-up service and performing in-house digitization to fulfill requests. Catalog records for many items will now include a button you can use to request pick-up of circulating items, or digitization of items.
+    Continuing for the Spring semester, access to Library spaces is available via our Study-Browse Service. Firestone and most branch libraries are also continuing to offer pick-up service and performing in-house digitization to fulfill requests. Catalog records for many items will now include a button you can use to request pick-up of circulating items, or digitization of items.
```
Comment (escowles) [inline app/views/catalog/_home_text.html.erb:40]: Can you make "Study-Browse Service" a link to https://library.princeton.edu/services/study-browse ?

File: app/views/catalog/_home_text.html.erb [hunk 2]
```diff
@@ -34,10 +34,10 @@
   </div>
   <div class="covid19-text">
     <%= link_to "https://library.princeton.edu/phased-planning-updates" do %>
-      <h3>New Services: Pick-up and Digitization</h3>
+      <h3>New Services: Study-Browse Service, Pick-up, and Digitization</h3>
     <% end %>
     <p>
-    Firestone and most branch libraries are now offering pick-up service and performing in-house digitization to fulfill requests. Catalog records for many items will now include a button you can use to request pick-up of circulating items, or digitization of items.
+    Continuing for the spring semester, access to Library spaces is available via our <%= link_to "https://library.princeton.edu/services/study-browse", "Study-Browse Service" %>. Firestone and most branch libraries are also continuing to offer pick-up service and performing in-house digitization to fulfill requests. Catalog records for many items will now include a button you can use to request pick-up of circulating items, or digitization of items.
```
Comment (escowles) [inline app/views/catalog/_home_text.html.erb:40]: I think the `link_to` is backwards because it's displaying the URL in staging...

**Comment by carolyncole** [comment]
@escowles the request you asked for are out on staging: https://catalog-staging.princeton.edu/

**Review by escowles — APPROVED** [review]
Thanks @carolyncole !

**Review by kevinreiss — APPROVED** [review]

**Pull request merged**